commit 5fd0dc735ee864ff1e19f7edc3bb98df6feeef72
Author: Gabor Karsay <gabor.karsay@gmx.at>
Date:   Tue Jul 11 20:36:18 2017 +0200

    [waveviewer] make sure color cache is updated once, fix #12

diff --git a/libparlatype/src/pt-waveviewer.c b/libparlatype/src/pt-waveviewer.c
index f5a36d9..39068f4 100644
--- a/libparlatype/src/pt-waveviewer.c
+++ b/libparlatype/src/pt-waveviewer.c
@@ -1125,6 +1125,13 @@ pt_waveviewer_set_wave (PtWaveviewer *self,
 	else
 		gtk_widget_set_size_request (self->priv->drawarea, data->length / 2, WAVE_MIN_HEIGHT);
 	gtk_adjustment_set_upper (self->priv->adj, data->length / 2);
+
+	/* It seems like the state_flags_changed and style_updated flags are
+	   never emitted under KDE/Plasma desktop or only once at the beginning
+	   when there is no parent window yet. If we don't manually update the
+	   cached style values, the colors are not set at all under KDE/Plasma. */
+	pt_waveviewer_update_cached_style_values (self);
+
 	gtk_widget_queue_draw (GTK_WIDGET (self));
 }
 
